activities/Clock.activity/locale.ini

```diff
@@ -213,6 +216,8 @@ TutoSetTimeTitle=Changer l'heure
 TutoSetTimeContent=Cliquez sur cette icône pour passer en mode changement d'heure. Dans ce mode vous pouvez déplacer les aiguilles pour indiquer l'heure que vous souhaitez
 TutoSetTimeGameTitle=Devinez l'heure
 TutoSetTimeGameContent=Cliquez sur cette icône pour passer au mode jeu. Dans ce mode, une heure est affichée au format numérique et vous devez déplacer les aiguilles pour les positionner sur l'heure correspondante
+TutoSetSecondsTitle = Peau de seconde main.
```

Comment:
Except if you're native from the target language, it's better to use the English localization everywhere. i.e. avoid automatic translation.

Reply:
So I type in the exact sentences in different languages as well? like the same which I wrote in English I should write in French ?

Reply:
Yes. There is a localization platform on https://translate.sugarizer.org where native speakers could translate strings without any development skill.

Reply:
I have done the changes you requested please review :)